src/blocks/Header/Header.tsx

```diff
@@ -130,12 +130,12 @@ export const HeaderBlock = (props: WithChildren<HeaderBlockFullProps>) => {
                                         <HTML>{title}</HTML>
                                     </h1>
                                     {description && (
-                                        <h5 className={b('description')}>
+                                        <div className={b('description')}>
```

Comment:
why?

Reply:
because 'h' tags are using for headers. but description is not header and this is error for html semantic. I have changed it to paragraph.